packages/now-cli/package.json

```diff
@@ -148,6 +148,7 @@
     "mri": "1.1.0",
     "ms": "2.1.1",
     "node-fetch": "1.7.3",
+    "now-client": "./packages/now-client",
```

Comment:
Similarly here.

Reply:
Adjusted this since it's in the PR scope. I'm not sure about the usage of all other dependencies, I think we can do it in a separate PR 🙌🏻